app/src/ereferrals/java/org/eyeseetea/malariacare/data/sync/importer/metadata/configuration/model/MetadataConfigurationsApi.java

```diff
@@ -0,0 +1,51 @@
+package org.eyeseetea.malariacare.data.sync.importer.metadata.configuration.model;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class MetadataConfigurationsApi {
```

Comment:
Wouldn't it better to name this MetadataConfiguration? This is a model representing a MetadataConfiguration, and not an API, right?

Reply:
I named it API because it represents a model from the API response, but I can change it as you suggest